lib/cmd/internal/messaging.js

```diff
@@ -31,50 +30,49 @@ function messaging(argv, cb) {
     case "ls":
       return topics(cb);
     case "topic":
-    {
-      if (args.length !== 1 && args.length !== 2) return unknown("topic", cb);
-      else {
-        if (args.length === 1) return topic(args[0], cb);
-        else return query_topic(args[0], args[1], cb);
+      if (args.length !== 1 && args.length !== 2) {
+        return unknown("topic", cb);
       }
-    }
+      if (args.length === 1) {
+        return topic(args[0], cb);
+      }
+      return query_topic(args[0], args[1], cb);
     case "message":
-    {
-      if (args.length !== 2) return unknown("message", cb);
-      else return message(args[0], args[1], cb);
-    }
-      ;
+      if (args.length !== 2) {
+        return unknown("message", cb);
+      }
+      return message(args[0], args[1], cb)
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)